models/message/index.js

```diff
@@ -1,7 +1,8 @@
 const path = require('path');
 
-const Model = require(path.resolve('./models/model')),
-      schema = require('./schema');
+const schema = require('./schema');
```

Comment:
:)
not a part of validation refactoring, is it?
Anyways thank you for trying to improve the code.